File: atinject-integration/src/main/resources/webapp/scripts/atinject/systemproperty/SystemProperty.js
define(["atinject/DTO"],
function(DTO){
    function SystemProperty() {
        this._class = "org.atinject.api.systemproperty.dto.SystemProperty"
    };

    SystemProperty.prototype = new DTO();

        SystemProperty.prototype.getKey = function() {
        return this.key;
    };

    SystemProperty.prototype.setKey = function(key) {
        this.key = key;
        return this;
    };

    SystemProperty.prototype.getValue = function() {
        return this.value;
    };

    SystemProperty.prototype.setValue = function(value) {
        this.value = value;
        return this;
    };


    
    return SystemProperty;
});

File: atinject-api-system-property/src/main/java/org/atinject/api/systemproperty/dto/SystemProperty.java
package org.atinject.api.systemproperty.dto;

import org.atinject.core.dto.DTO;

public class SystemProperty extends DTO {

    private static final long serialVersionUID = 1L;

    private String key;
    private String value;

    public String getKey() {
        return key;
    }

    public void setKey(String key) {
        this.key = key;
    }

    public SystemProperty withKey(String key) {
        setKey(key);
        return this;
    }

    public String getValue() {
        return value;
    }

    public void setValue(String value) {
        this.value = value;
    }

    public SystemProperty withValue(String value) {
        setValue(value);
        return this;
    }
}
